ensime_shared/ensime.py

```diff
@@ -1,24 +1,30 @@
 # coding: utf-8
+"""
+The Python core of the ensime-vim plugin, which bridges between ENSIME
+client/server communication and Vim as the UI, event, and scripting layer.
+"""
 
 import os
 
 from .client import EnsimeClientV1, EnsimeClientV2
-from .config import ProjectConfig
+from .config import feedback, ProjectConfig
 from .editor import Editor
-from .launcher import EnsimeLauncher
+from .errors import InvalidJavaPathError
+from .server import Server
 
 
-def execute_with_client(quiet=False,
-                        bootstrap_server=False,
-                        create_client=True):
-    """Decorator that gets a client and performs an operation on it."""
+def execute_with_client():
```

Comment:
Yeah these args and the related `EnsimeClient.setup` method, probably my happiest moments of making these changes 😁  🍾